sdk/core/azure-core/src/main/java/com/azure/android/core/util/CoreUtils.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.core.util;
+
+/**
+ * This class contains utility methods useful for building client libraries.
+ */
+public class CoreUtils {
```

Comment:
Should mark this `final` as well. OR, what about using an interface with static methods (since the Android toolchain does now support these on all API levels)?

Reply:
Changing it to an interface with static methods after our in-person discussion.